tensorflow/compiler/xla/service/gpu/cudnn_conv_rewriter.cc

```diff
@@ -521,6 +521,14 @@ StatusOr<bool> RunOnInstruction(HloInstruction* conv) {
 
     // If all else fails, try a forward convolution.
     if (CanImplementAsCudnnForwardConv(conv)) {
+      if (primitive_util::IsIntegralType(
```

Comment:
This is where we disallow int8-to-int8 convolution for CuDNN mapping.
Make it a fatal error, because int8-to -int8 convolution semantics is not implemented.